package.json

```diff
@@ -72,6 +72,7 @@
     "prettier": "^2.8.7"
   },
   "dependencies": {
-    "cross-env": "^7.0.3"
+    "cross-env": "^7.0.3",
+    "eslint-plugin-unused-imports": "^2.0.0"
```

Comment:
I guess it would be beneficial to have dependencies at the root of the monorepo, as they would be applicable to the entire project. Otherwise, I will have to add them separately to the individual apps within the project.

Reply:
Its okay to add them to each project, we would like each project to work independently

Reply:
ok, going to add them separately now

Reply:
done. Can you have a look now? Thank you

Reply:
Sure. No worries at all
Done.